119 continue translations

mabelene opened the pull request:
@LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code

LemonFace0309 commented:
> @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code

wait wdym by typography changes? Lol

LemonFace0309 commented on components/Dashboard/Modals/ProfileDialog/AccountDetails.js:
```diff
@@ -96,10 +99,10 @@ const AccountDetails = ({ session, editMode, saveChanges, setSaveChanges, setEdi
         if (!passed) {
-          alert(message ?? 'You account info has been updated sucessfully 😃');
+          alert(message ?? t('SUCCESS_CHANGE_PROFILE'));
           setEditMode(true);
         } else {
-          alert(message ?? 'Internal Server Error');
+          alert(message ?? t('ERROR_SERVER'));
```
can you prefix with `LABEL_` ?

mabelene replied:
okk do u want all translations to be prefixed with LABEL_ ?

LemonFace0309 replied:
Yes pls :3

LemonFace0309 commented on components/Dashboard/Modals/ProfileDialog/ChangePassword.js:
```diff
@@ -76,13 +78,13 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
       <Grid item xs={12} md={6}>
         <Typography variant="h6" className={classes.title}>
-          Enter your current and new password
+          {t('PROMPT_PASSWORD')}
```
same here

LemonFace0309 commented on components/Dashboard/Modals/ProfileDialog/ChangePassword.js:
```diff
@@ -54,20 +56,20 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
     return (
       <Typography variant="h6" className={classes.title}>
-        Cannot change password for users signed up through third party services like Google or Facebook.
+        {t('ERROR_THIRD_PARTY_PASSWORD')}
```
same here

LemonFace0309 commented on components/Dashboard/Modals/ProfileDialog/ChangePassword.js:
```diff
@@ -54,20 +56,20 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
     } catch (err) {
       setServerError(true);
-      alert(err?.response?.data?.message ?? 'Internal Server Error');
+      alert(err?.response?.data?.message ?? t('ERROR_SERVER'));
```
same here

mabelene commented:
> @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code



> > @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code
> 
> @mabelene wait wdym by typography changes? Lol

@LemonFace0309 oops mb, i meant translations

LemonFace0309 commented on components/Dashboard/Modals/ProfileDialog/ChangePassword.js:
```diff
@@ -54,20 +56,20 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
       });
       console.debug(response);
-      alert(response?.data.message ?? 'Password updated succesfully 😃');
+      alert(response?.data.message ?? t('SUCCESS_UPDATE_PASSWORD'));
```
same here

LemonFace0309 commented on components/Dashboard/Sidebar.js:
```diff
@@ -1,9 +1,10 @@
 /* eslint-disable jsx-a11y/accessible-emoji */
+import { useTranslation } from 'next-i18next';
+
```
delete line 3 pls

LemonFace0309 reviewed (approved):
LGTM, just make changes I suggested, click on resolve conversation when you're done, and ensure it works properly for different locales. :D

LemonFace0309 commented:
> > @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code
> 
> > > @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code
> > 
> > 
> > @mabelene wait wdym by typography changes? Lol
> 
> @LemonFace0309 oops mb, i meant translations

if you want. I don't mind. translations are ezpz to review. Just rerequest a review from me if you do. 
![image](https://user-images.githubusercontent.com/65472533/130149938-7e8a54f1-656a-4b76-ac6d-d9cc7573517b.png)
(the circle icon besides the green checkmark)
